src/components/fileviewer.js

```diff
@@ -49,12 +54,43 @@ export default class FileViewerContainer extends Component {
       "format": "list"
     })
     .then(data => {
-      this.setState({coverage: data});
       // TODO remove these log lines
       console.log(data);
+      this.setState({coverage: data});
+      this.parseTestsCoverage(data.data);
     });
   }
 
+  /* Parse data returns from ActiveData */
+  parseTestsCoverage(data) {
+    console.log(data.length);
+    var stat = []
+
+    for (var i = 0; i < data.length; i++) {
```

Comment:
Please use `forEach`, like so:

    data.forEach((d, i) => {  
       var coveredLines = d.source.file.covered;

you may not even need `i`